src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,126 @@
 package co.inventorsoft.academy.collections.model;
 
+
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
 
+    @Override
+    public String toString() {
+        return "Range{" +
+                "expectedElements=" + expectedElements +
+                '}';
+    }
+
+    public final List<T> expectedElements = new ArrayList<>();
+
+    public static class IntegerNextValue implements Function<Integer, Integer> {
+
+        @Override
+        public Integer apply(Integer i) {
+            return i + 1;
+        }
+    }
+
+
+    public static Range<Integer> of(Integer first, Integer last) {
+        IntegerNextValue inv = new IntegerNextValue();
+        Range<Integer> rangeOf = Range.of(first, last, inv);
```

Comment:
You don't use rangeOf variable, return value immediately

Reply:
ok